src/lua/error.lua

```diff
@@ -143,6 +144,14 @@ local function error_unpack(err)
         local f = fields[i]
         result[ffi.string(f._name)] = mp_decode(f._data)
     end
+    -- Hide redundant fields from unpack output (gh-9101).
+    if compat.box_error_unpack_type_and_code.current == 'new' then
+        result.custom_type = nil
+        result.base_type = nil
+        if result.code == 0 then
+            result.code = nil
+        end
```

Comment:
AFAIR, the idea is to make it more concise in a console output/in a log message. IOW, it should affect serialization.

Am I understand right that it will become true after PR #9716?

Reply:
No, this one is independent. Instead the error serialization done in #9716 depends on this one as it uses `error:unpack()`.

Reply:
That's what I meant. Thanks for clarification!